broker/src/main/java/org/apache/rocketmq/broker/schedule/ScheduleMessageService.java

```diff
@@ -218,8 +218,7 @@ public String encode() {
 
     @Override
     public boolean load() {
-        boolean result = super.load();
-        result = result && this.parseDelayLevel();
```

Comment:
If super.load() is not called, the JSON file will not be parsed, even during startup. IMO, we can add a new method called loadWhenSyncDelayOffset, which will not correct the delay offset when syncDelayOffset is called.

Reply:
I added a `loadWhenSyncDelayOffset` method, called at `this.brokerController.getScheduleMessageService().loadWhenSyncDelayOffset();`. `loadWhenSyncDelayOffset `doesn't read `delayOffset.json` repeatedly.

Reply:
IMO, we should read delayOffset.json and should not correct the delay offset when slave broker sync delay offset.

Reply:
I don't know if I understand correctly, but we correct the offset when we initialize the startup, not when we synchronize it

Reply:
This doesn't feel right either.